docs/src/main/asciidoc/security-openid-connect-concept.adoc

```diff
@@ -3,21 +3,24 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Using OpenID Connect (OIDC) to Protect Service Applications using Bearer Token Authorization
-include::_attributes.adoc[]
+[id="security-oidc-bearer-authentication-concept"]
+= OIDC Bearer authentication
+include::./attributes.adoc[]
 :categories: security
-:summary: This guide demonstrates how your Quarkus application can use Keycloak to protect your JAX-RS applications using bearer token authorization, where these tokens are issued by a Keycloak server.
+:toc:
```

Comment:
In the new Quarkus _diataxis_ asciidoc file header, we no longer need to add `:toc:` to each file. If it gets added, there will be duplicate tables of contents on the page in the portal.

The `:toc:` setting was required in the pre-diataxis implementation. Itt was taken out of individual docs in the recent switch over to diataxis framework and new home page. The toc is now set dynamically through the presence of the attribute file include on line 8.

I added content about this to the contributor guide updates in PR #30043, which is almost ready for review.

@ebullient - Did I miss a key point here?

Reply:
Removed. Thanks!